c_src/dpiStmt_nif.h

```diff
@@ -45,7 +46,8 @@ extern DPI_NIF_FUN(stmt_close);
         IOB_NIF(stmt_getQueryInfo, 2),       \
         IOB_NIF(stmt_getQueryValue, 2),      \
         IOB_NIF(stmt_getNumQueryColumns, 1), \
-        DEF_NIF(stmt_close, 2)
+        DEF_NIF(stmt_close, 2),              \
+        DEF_NIF(stmt_getInfo, 1)
```

Comment:
check for server trip (DEF/IOB)!

Reply:
changed to IOB because there's a conditional, underlying `OCIAttrGet` call that does 0-2 round trips